libs/sqlite3/Makefile

```diff
@@ -67,11 +75,20 @@ $(call Package/sqlite3/Default/description)
  formats.
 endef
 
+
```

Comment:
gratuitous whitesspace change?

Reply:
indeed

Reply:
My bad; fixed in https://github.com/openwrt/packages/commit/25757a96fa804241ea04aaae9006efc9aa24648e

Reply:
heh, well, I wouldn't have necessarily gone and made another commit, but ok :)